src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -800,15 +805,48 @@ class EntityApiServiceSpec extends ApiServiceSpec {
         assertResult(StatusCodes.OK) {
           status
         }
+
+        def attrToString(a: Attribute): String = {
+          a match {
+            case AttributeString(value) => value
+            case AttributeNumber(value) => value.toString
+            case AttributeValueList(list) if list.isEmpty => ""
+            case AttributeValueList(list) => attrToString(list.head)
+            case _ => throw new RawlsException("make the compiler stop whining")
+          }
+        }
+
         assertResult(EntityQueryResponse(
           defaultQuery.copy(sortField = "mixed", pageSize = paginationTestData.numEntities),
           EntityQueryResultMetadata(paginationTestData.numEntities, paginationTestData.numEntities, 1),
-          paginationTestData.entities.sortBy(_.attributes("mixed") match {
-            case AttributeString(value) => value
-            case AttributeNumber(value) => value.toString
+          paginationTestData.entities.sortBy( e => attrToString(e.attributes("mixed")) ))) {
+          responseAs[EntityQueryResponse]
```

Comment:
I don't see how this test checks that `paginationTestData` is sorted, just that the response is a valid one.